include/filewriter/crc.hpp

```diff
@@ -0,0 +1,16 @@
+/*
+**  CRC.H - header file for SNIPPETS CRC and checksum functions
```

Comment:
Did you write this/these files?

Reply:
/* Copyright (C) 1986 Gary S. Brown.  You may use this program, or
   code or tables extracted from it, as desired without restriction.*/

-> https://web.mit.edu/freebsd/head/sys/libkern/crc32.c